ibis/backends/oracle/__init__.py

```diff
@@ -274,6 +274,11 @@ def raw_sql(self, query: str | sg.Expression, **kwargs: Any) -> Any:
         con = self.con
         cursor = con.cursor()
 
+        # TODO(deepyaman): Fix Oracle MERGE query generation in SQLGlot.
+        if "MERGE" in query:
+            assert "AS " in query
+            query = query.replace("AS ", "")
+
```

Comment:
This is obviously unacceptable, but will wait on https://github.com/tobymao/sqlglot/issues/5910 to try and figure out what's the best way to work arounds this (unless somebody else has ideas).

Reply:
Fixed here https://github.com/tobymao/sqlglot/pull/5911, will release soon.

Reply:
@georgesittas Tested locally with SQLGlot 27.16.3, and removing this hack works perfectly. Thank you!!

(So impressed by the 0-issue inbox on SQLGlot BTW.)